website/conferences/views.py

```diff
@@ -138,6 +155,27 @@ def add_poster_by_email(conference, message):
     if node_created and user_created:
         signals.osf4m_user_created.send(user, conference=conference, node=node)
 
+    # Send log to Keen
+    keen.add_event('osf4m', {
+        "user": {
+            "username": user.username,
+            "userEmail": message.sender_email,
+            "locale": user.locale,
+            "urls": {
+                "profile": "/{}/".format(user._id),
+                "api": "/api/v1/profile/{}/".format(user._id)
+            },
+            "timezone": user.timezone,
+            "fullname": user.fullname,
+            "isActive": user.is_active,
+            "id": user._id
+        },
+        "meeting": message.conference_name,
+        "userCreated": user_created,
+        "timeSubmitted": _dt_to_iso8601(datetime.utcnow())
```

Comment:
Can you make the above three attributes under an "osf4m" top-level key?  Could you also add a static field product.id?  `"product": { "id": "osf4m" }`